src/features/import/components/ImportDialog/Configure/Configuration/OrgConfig.tsx

```diff
@@ -31,9 +33,19 @@ const OrgConfig: FC<OrgConfigProps> = ({ uiDataColumn }) => {
       padding={2}
       sx={{ overflowY: 'auto' }}
     >
-      <Typography sx={{ paddingBottom: 2 }} variant="h5">
-        <Msg id={messageIds.configuration.configure.orgs.header} />
-      </Typography>
+      <Box alignItems="baseline" display="flex" justifyContent="space-between">
+        <Typography sx={{ paddingBottom: 2 }} variant="h5">
+          <Msg id={messageIds.configuration.configure.orgs.header} />
+        </Typography>
+        <Button
+          onClick={() => {
+            guessOrgs();
+          }}
+        >
+          {messages.configuration.configure.orgs.guess().toLocaleUpperCase()}
```

Comment:
I am unsure about the `toLocaleUpperCase()` here. I think the button already applies that type of text transform. In the future, we are probably moving away from uppercase button labels, so I don't want to hardcode uppercase here because it would stay uppercase even when we get around to making that theme change.